src/components/header/ControlOption.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function ControlOption(props){
+
+    const { item } = props;
```

Comment:
You can check for how to destructure nested objects, though this is correct. But we can do it in a single line as the objects are not much complex